src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -164,7 +166,7 @@ export default class OverflowMenu extends Component {
             description={iconDescription}
             style={{ width: '100%' }}
           />
-          {floatingMenu ? (
+          {!open ? null : floatingMenu ? (
```

Comment:
Could we extract the ternary logic into a function for readability?

Reply:
Ooops just realized I merged this before this comment was addressed 😢 I can revert it if you want?!

Reply:
No worries @marijohannessen! 🙂

@joshblack Thank you for giving it a thought, did some clean-up there: https://github.com/carbon-design-system/carbon-components-react/pull/465